### DISCUSSION TITLE
Try to fixed login using github account/email

### DISCUSSION THREAD

**Discussion by nuhptr** [vercel/community#4304] [Account Issue] [answered]
### Question

I have tried to delete my Vercel account and create a new one using email and already emailed support, but I haven't received a response (case number 00162564)

**Comment by amyegan** [comment, marked as answer]
Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases before they reach yours. We all appreciate your patience while the team works through an unusually high number of tickets.

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Reply by nuhptr** [reply]
I have waited for more than 5 days and no response
what happened?
